src/test/soundproxy_test.cpp

```diff
@@ -21,10 +21,10 @@ class SoundSourceProxyTest : public MixxxTest {
 TEST_F(SoundSourceProxyTest, ProxyCanOpen) {
     // This test piggy-backs off of the cover-test files.
     const QString kCoverFilePath(
-            QDir::currentPath() + "/src/test/id3-test-data/cover-test.");
+            QDir::currentPath() + "/src/test/id3-test-data/cover-test");
 
     QStringList extensions;
-    extensions << "aiff" << "flac" << "mp3" << "ogg" << "wav";
+    extensions << ".aiff" << ".flac" << "-png.mp3" << ".ogg" << ".wav";
```

Comment:
Yeah it is enough to have one mp3 file for to test if the soundsourceproxy can read mp3 files correctly.